jaeger-core/src/main/java/io/jaegertracing/internal/JaegerTracer.java

```diff
@@ -315,7 +320,7 @@ private JaegerSpanContext createNewContext(String debugId) {
         }
       }
 
-      return new JaegerSpanContext(id, id, 0, flags);
+      return JaegerTracer.this.objectFactory.createSpanContext(id, id, 0, flags, Collections.<String, String>emptyMap(), debugId);
```

Comment:
feels like it could use a helper function, if only to avoid this `JaegerTracer.this.objectFactory.createSpanContext` verbosity

Reply:
OK makes sense.